knowledge-base/images-charts-barcodes-and-shapes-are-not-being-displayed-in-the-silverlight-report-viewer.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Images, Charts, Barcodes and Shapes are not being displayed in the Silverlight report viewer.
+description: Images, Charts, Barcodes and Shapes are not being displayed in the Silverlight report viewer.
+type: how-to
+page_title: Images, Charts, Barcodes and Shapes are not being displayed in the Silverlight report viewer.
+slug: images-charts-barcodes-and-shapes-are-not-being-displayed-in-the-silverlight-report-viewer
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting Silverlight</td>
```

Comment:
What is this product? Is this the viewer for Silverlight? If so, mark it as it is mentioned in the docs and on marketing pages so that we keep things consistent. 

Also are there other things we can add in the environment table? Like version of reporting, or .net?